README.md

```diff
@@ -141,49 +152,55 @@ is 'active' and may be implemented with the goal of eventual inclusion
-cases the topic may be discussed at the larger [weekly meeting]. In
-either case a summary from the meeting will be posted back to the RFC
-pull request.
-* Once both proponents and opponents have clarified and defended
-positions and the conversation has settled, the shepherd will take it
-to the [core team] for a final decision.
-* Eventually, someone from the [core team] will either accept the RFC
-by merging the pull request, assigning the RFC a number (corresponding
-to the pull request number), at which point the RFC is 'active', or
-reject it by closing the pull request.
+stakeholders to discuss the issues in greater detail.
+* The sub-team will discuss the RFC PR, as much as possible in the comment
+thread of the PR itself. Offline discussion will be summarized on the PR comment
+thread.
+* Once both proponents and opponents have clarified and defended positions and
+the conversation has settled, the RFC will enter its *final comment period*
+(FCP). This is a final opportunity for the community to comment on the PR and is
+a reminder for all members of the sub-team to be aware of the RFC.
+* The FCP lasts one week. It may be extended if consensus between sub-team
+members cannot be reached. At the end of the FCP,  the [sub-team] will either
+accept the RFC by merging the pull request, assigning the RFC a number
+(corresponding to the pull request number), at which point the RFC is 'active',
+or reject it by closing the pull request. How exactly the sub-team decide on an
+RFC is up to the sub-team.
+
 
 ## The role of the shepherd
 [The role of the shepherd]: #the-role-of-the-shepherd
 
-During triage, every RFC will either be closed or assigned a shepherd.
-The role of the shepherd is to move the RFC through the process. This
-starts with simply reading the RFC in detail and providing initial
-feedback. The shepherd should also solicit feedback from people who
-are likely to have strong opinions about the RFC. Finally, when this
-feedback has been incorporated and the RFC seems to be in a steady
-state, the shepherd will bring it to the meeting. In general, the idea
-here is to "front-load" as much of the feedback as possible before the
-point where we actually reach a decision.
+During triage, every RFC will either be closed or assigned a shepherd from the
+relevant sub-team. The role of the shepherd is to move the RFC through the
+process. This starts with simply reading the RFC in detail and providing initial
+feedback. The shepherd should also solicit feedback from people who are likely
+to have strong opinions about the RFC. When this feedback has been incorporated
+and the RFC seems to be in a steady state, the shepherd and/or sub-team leader
+will announce an FCP. In general, the idea here is to "front-load" as much of
+the feedback as possible before the point where we actually reach a decision -
+by the end of the FCP, the decision on whether or not to accept the RFC should
```

Comment:
> by the end of the FCP, the decision on whether or not to accept the RFC should be obvious from the RFC discussion thread.

That seems a bit strong -- there have been and will be cases where there isn't an on-thread consensus, but the arguments on both sides have been exhausted and we just need to make a decision.